rewrite-gradle/src/main/java/org/openrewrite/gradle/RemoveEnableFeaturePreview.java

```diff
@@ -0,0 +1,69 @@
+package org.openrewrite.gradle;
+
+import java.util.List;
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+import org.jetbrains.annotations.NotNull;
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Option;
+import org.openrewrite.Preconditions;
+import org.openrewrite.Recipe;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.groovy.GroovyIsoVisitor;
+import org.openrewrite.java.tree.Expression;
+import org.openrewrite.java.tree.J;
+import org.openrewrite.java.tree.J.MethodInvocation;
+
+@Value
+@EqualsAndHashCode(callSuper = false)
+public class RemoveEnableFeaturePreview extends Recipe {
+
+  private static final String ENABLE_FEATURE_PREVIEW_METHOD_NAME = "enableFeaturePreview";
+
+  @Option(displayName = "The feature preview name",
+      description = "The name of the feature preview to remove.",
+      example = "ONE_LOCKFILE_PER_PROJECT"
+  )
+  public String previewFeatureName;
+
+  @Override
+  public @NotNull String getDisplayName() {
+    return "Remove a Gradle enableFeaturePreview method";
+  }
+
+  @Override
+  public @NotNull String getDescription() {
+    return "Remove a Gradle enableFeaturePreview method from settings.gradle / settings.gradle.kts.";
```

Comment:
```suggestion
    return "Remove an enabled Gradle preview feature from `settings.gradle`.";
```

Since you're using a `GroovyIsoVisitor` this only works with Gradle Groovy DSL. Gradle Kotlin DSL is still a work in progress and not yet available.